ocsf_validator/type_mapping.py

```diff
@@ -9,8 +9,6 @@
     VersionMatcher(),
     DictionaryMatcher(),
     CategoriesMatcher(),
-    IncludeMatcher(),
-    CategoriesMatcher(),
```

Comment:
`note`: unrelated, but I noticed `CategoriesMatcher` appeared twice in this list.